Update Order Page View.md (#168)
PrestaShop/prestashop-specs: back-office/orders/orders/Order Page View.md

It displays the date and from where the user came from.

## VII. Links

## VIII. Multistore

> An order from a specific shop can be edited from any shops. Even if they are separeted from differents groups.

Reviewer: ```suggestion
An order made from a specific shop can be edited from any shops. Even if the shop where the order is edited isn't in the same group as the shop where the order was made.
```

> In a multistore context, the next order and previous buttons work depending on the view of the shop or the group of shops the Back-Office is set.

Reviewer: ```suggestion
In a multistore context, the next and previous buttons for navigating between orders work according to the selected context in the back-office (specific shop, group of shops or all shops)
```

> If the order listing is filtered by a multistore group view then the user can navigate between orders in the order view page only from this shop group.

Reviewer: ```suggestion
If a specific shop group is selected as context in the order list, then the user can navigate between orders belonging only to this shop group in the order view page.
```

> If the order listing is filtered by a specific shop from the multistore setting then the user can navigate between orders in the order view page only from this shop.

Reviewer: ```suggestion
If a specific shop is selected as context in the order list, then the user can navigate between orders belonging only to this shop in the order view page.
```

> And if the order listing is set in "All shop" in the multistore setting then the user can navigate between orders in the order view page from all the shops.

Reviewer: ```suggestion
If "All shops" is selected as context in the order list, then the user can navigate between orders from all the shops in the order view page.
```
